### PR TITLE
fix(TPSVC-18473): update exception handling

### PR DESCRIPTION
**Pull request by mbutko**
**What is the problem this Pull Request is trying to solve?**
 
**What is the chosen solution to this problem?**
 
**Link to the JIRA issue**

 
**Please check if the Pull Request fulfills these requirements**
- [ ] The PR commit message follows our [guidelines](https://github.com/Talend/daikon/blob/master/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features, coverage should be over 75% in the new code)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in JIRA), if any, are all linked or available in the Pull Request

 
**[ ] This Pull Request introduces a breaking change**

### CODE REVIEW

**Review by sgandon — APPROVED** [review]
LGTM

**Comment by sgandon** [comment]
This looks like a breaking change, please make sure to communicate around that change during the release process and may be change the major item of the daikon version.

**Comment by sgandon** [comment]
that would also be nice to have some unit test for this.

File: daikon-spring/daikon-spring-auth/pom.xml [hunk 1]
```diff
@@ -89,30 +84,41 @@
...
+<!--                <plugin>-->
+<!--                    <groupId>org.apache.maven.plugins</groupId>-->
+<!--                    <artifactId>maven-javadoc-plugin</artifactId>-->
+<!--                    <configuration>-->
+<!--                        <skip>true</skip>-->
+<!--                    </configuration>-->
+<!--                </plugin>-->
+<!--            </plugins>-->
+<!--        </pluginManagement>-->
+<!--    </build>-->
```
Comment (jehervy) [inline daikon-spring/daikon-spring-auth/pom.xml:123]: Shouldn't we remove this whole part ?

**Review by jehervy — APPROVED** [review]
LGTM

**Pull request merged**